test/src/runner/cli/testCliRunner.js

```diff
@@ -485,6 +485,33 @@ module.exports = {
       }, 'No testing environment specified.');
     },
 
+    testParseTestSettingsNull : function() {
+      mockery.registerMock('fs', {
+        statSync : function(module) {
+          if (module == './null.json') {
+            return {isFile : function() {return true;}};
+          }
+          throw new Error('Does not exist');
+        }
+      });
+
+      mockery.registerMock('../../util/logger.js', {
+        error : function() {
+          assert.ok(false, 'Logger.error should not be called');
+        }
+      });
+
+      var CliRunner = common.require('runner/cli/clirunner.js');
+      assert.throws(function() {
```

Comment:
It checks if the code in the try block in lib/runner/cli/clirunner.js:100 does **not** get executed. Just because a null gets given the code in the catch should be completly executed without an error. But the error _No testing environment specified_ should be thrown, because it is not given.

If wanted I can get in more detail, just let me know.